From 97276ecb618954514ea4526243d7379b681632d6 Mon Sep 17 00:00:00 2001
From: Xi Ruoyao <xry111@xry111.site>
Date: Mon, 23 Jun 2025 18:59:39 +0800
Subject: [PATCH 13/63] loonggpu: Remove redundant info->par assignment

It's already assigned by lg_drm_fb_helper_fill_info (no matter what the
kernel version is).

Signed-off-by: Xi Ruoyao <xry111@xry111.site>
---
 loonggpu/loonggpu_fb.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/loonggpu/loonggpu_fb.c b/loonggpu/loonggpu_fb.c
index ad0fcfb..c0cca28 100644
--- a/loonggpu/loonggpu_fb.c
+++ b/loonggpu/loonggpu_fb.c
@@ -219,7 +219,6 @@ static int loonggpufb_create(struct drm_fb_helper *helper,
 		goto out;
 	}
 
-	info->par = helper;
 	info->skip_vt_switch = false;
 
 	fb = kzalloc(sizeof(struct loonggpu_framebuffer), GFP_KERNEL);
-- 
2.52.0

